bindings/python/README.md

```diff
@@ -130,4 +130,757 @@ def main():
+| --------------------- | -------------------------------------------- | ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [network]             | <code>str</code>                             | <code>undefined</code> | The network                                                                                                                                                       |
+| [node]                | <code>str</code>                             | <code>undefined</code> | A node URL                                                                                                                                                        |
+| [nodes]               | <code>list[str]</code>                       | <code>undefined</code> | An array of node URLs                                                                                                                                             |
+| [node_sync_interval]  | <code>int</code>                             | <code>undefined</code> | The interval for the node syncing process                                                                                                                         |
+| [node_sync_disabled]  | <code>bool</code>                            | <code>undefined</code> | Disables the node syncing process. Every node will be considered healthy and ready to use                                                                         |
+| [node_pool_urls]      | <code>str</code>                             | <code>undefined</code> | An array of node pool URLs                                                                                                                                        |
+| [request_timeout]     | <code>int</code>                             | <code>undefined</code> | Sets the default HTTP request timeout                                                                                                                             |
+| [api_timeout]         | <code>dict</code>                            | <code>undefined</code> | The API to set the request timeout. Key: 'GetHealth', 'GetInfo', 'GetPeers', 'GetTips', 'PostMessage', 'GetOutput', 'GetMilestone' Value: timeout in milliseconds |
+| [local_pow]           | <code>bool</code>                            | <code>undefined</code> | Flag determining if PoW should be done locally or remotely                                                                                                        |
+| [tips_interval]       | <code>int</code>                             | <code>undefined</code> | Tips between tips                                                                                                                                                 |
```

Comment:
Maybe `Time between requests for new tips during PoW`?

Reply:
Agree, thanks for identifying this.